### PR TITLE
15687 - Pay remove restx

### PR DESCRIPTION
**Pull request by ochiu**
*Issue #:*
https://github.com/bcgov/entity/issues/15687

*Description of changes:*
- remove restx from pay-api

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-pay license (Apache 2.0).

### CODE REVIEW

File: pay-api/src/pay_api/resources/ops.py
Comment (Jxio) [inline pay-api/src/pay_api/resources/ops.py:1]: can ops.py move to ../v1/ ?

Comment (ochiu) [reply pay-api/src/pay_api/resources/ops.py:1]: The ops endpoint is part of the /ops route and not under /api/v1, so I kept it out of the v1 folder as it is not specific to a version.

**Comment by seeker25** [comment]
Please move `git mv` to preserve the history of the files.

**Review by seeker25 — CHANGES_REQUESTED** [review]
Please use `git mv` to preserve the history of the files. This is more important in PAY than AUTH.

**Comment by seeker25** [comment]
After reviewing with Ody, `git mv` wasn't the issue. It's done in the commits

**Comment by seeker25** [comment]
Another thing to note we discovered: when moving these files - better not to Squash and merge as is

**Review by seeker25 — APPROVED** [review]

**Pull request merged**